pkg/api/config.go

```diff
@@ -73,6 +76,12 @@ func (c *Config) Validate() error {
 	merr = merr.Append(checkHost(c.Host))
 	merr = merr.Append(c.VerboseSettings.Validate())
 
+	_, apikeyPtr := c.AuthWriter.(*auth.APIKey)
+	_, apikey := c.AuthWriter.(auth.APIKey)
+	if c.Host == ESSEndpoint && !(apikey || apikeyPtr) {
```

Comment:
I'm not too sure the client should be handling this 🤔 But then again, is the API returning an error that makes sense in this case?

Reply:
The problem with not handling this in the client is that we'll end up with consumers reporting that their `username`  / `password` credentials return `401` when using against ESS 😄